### PR TITLE
Update ug specific instructions

### PR DESCRIPTION
**Pull request by jannaleong**

### CODE REVIEW

**Review by yleeyilin — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -648,47 +682,94 @@ Examples:
...
+* `/remind`
+  <br>
+  The above command displays all contacts with note deadlines from today onwards.
+
+[//]: # (insert image showing example)
+
 <div markdown="span" class="alert alert-primary">:bulb: **Tips:**<br>
 
-* To see help for all commands, the corresponding command field is `general`.<br>
+* Specifying '/remind [any text]' is equivalent to '/remind'<br>
```
Comment (jamessinmaojun) [inline docs/UserGuide.md:767]: period

File: docs/UserGuide.md [hunk 2]
```diff
@@ -648,47 +682,94 @@ Examples:
...
+* To return back to full contact list, use the command '/list'
```
Comment (jamessinmaojun) [inline docs/UserGuide.md:768]: period

File: docs/UserGuide.md [hunk 3]
```diff
@@ -648,47 +682,94 @@ Examples:
...
+<div markdown="span" class="alert alert-primary">:bulb: **Tips:**<br>
+
+* If you wish to get help for all commands, enter the command input as general to be directed to a general help window.<br>
+
+</div>
+
+<div markdown="span" class="alert alert-danger">:x: **Error Handling:**<br>
+
+* If you receive the error message "Failed to give help - Help requires a command field. 🐾",
+  ensure you have specified a valid command in the command field <br>
```
Comment (jamessinmaojun) [inline docs/UserGuide.md:748]: period

File: docs/UserGuide.md [hunk 4]
```diff
@@ -516,48 +517,81 @@ More Examples:
...
+
+</div>
+
+
+<div markdown="span" class="alert alert-danger">:x: **Error Handling:**<br>
+
+* If you receive the error message "Failed to add note to Pooch Contact - Name is not specified 🐾",
+  ensure you have specified a name in the name field <br>
+* If you receive the error message "Failed to add note to Pooch Contact - Name does not exist in our address book 🐾",
+  ensure you have specified a name of a contact that is already exists in the planner <br>
```
Comment (jamessinmaojun) [inline docs/UserGuide.md:588]: period

File: docs/UserGuide.md [hunk 5]
```diff
@@ -516,48 +517,81 @@ More Examples:
...
+* Notes added are intended to be short details! <br>
+* To remove a note, specify in the note field "No note here". This note is case-sensitive.
+
+</div>
+
+
+<div markdown="span" class="alert alert-danger">:x: **Error Handling:**<br>
+
+* If you receive the error message "Failed to add note to Pooch Contact - Name is not specified 🐾",
+  ensure you have specified a name in the name field <br>
```
Comment (jamessinmaojun) [inline docs/UserGuide.md:586]: period

**Review by jamessinmaojun — APPROVED** [review]
lgtm

**Pull request merged**